docs/tutorial/using-docker-container.md

```diff
@@ -76,19 +72,12 @@ at last, to check if the commands have taken effect with `wsl --status` command.
 
 ## Docker Desktop for Windows
 
-As the distribution Ubuntu 20.04 has been updated to version 2, so it needs to modify accordingly from docker side and choose the Ubuntu 20.04 as the default WSL integration as well.
+As the distribution Ubuntu has been updated to version 2, so it needs to modify accordingly from docker side and choose the Ubuntu as the default WSL integration as well.
```

Comment:
I would rephrase this to:

As the Ubuntu distribution has been updated to version 2, adjustments on the Docker side are required, and Ubuntu should also be chosen as the default WSL integration.